packages/tasit-action/src/TasitAction.js

```diff
@@ -168,22 +168,29 @@ class TransactionSubscription extends Subscription {
           this.#tx.hash
         );
 
-        if (receipt !== null) {
-          this.#txConfirmed = true;
-        } else {
-          if (this.#txConfirmed)
-            this._emitErrorEvent(
-              new Error(`Your message has been included in an uncle block.`),
-              eventName
-            );
+        const includedInAnUncle =
```

Comment:
Nice variable to define this clearly

Reply:
Would `blockReorgOccurred` be a better name than `includedInAnUncle` or no?

Reply:
Yes, better.

Reply:
Resolving this